pages/components/card/card.css

```diff
@@ -7,23 +8,50 @@
 	justify-content: center;
 	width: 35rem;
 	border-radius: 5px;
-	background-color: var(--colour-black-fourth);
-	border: 2px solid var(--colour-text-grey);
+	background-color: var(--colour-black-fifth);
```

Comment:
I would suggest use number in number like colour-black-5 but maybe again its personal preference. Rest all good I think by me.